docs/docs/04-standard-library/01-cloud/endpoint.md

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
The `.md` file should be created in the https://github.com/winglang/wing/blob/main/libs/wingsdk/src/cloud/ folder, just like the other cloud resources